cpu/samd21/ldscripts/samr21x18a.ld

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2015 Freie Universität Berlin
+ * Copyright (C) 2016 Freie Universität Berlin
```

Comment:
IIRC the copyright should (must?) always be extended, not modified. I.e. it should be `Copyright (C) 2015, 2016 ...` instead.